git-hub

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+import subprocess
+import github
+import yaml
+import click
+import sys
+
+
+def pr(num):
```

Comment:
Add docstring in numpydoc format.  See `numpy` or `scikit-image` or `scikit-learn` for how those typically look, or see https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

Reply:
Is the docstring for every function or for the file or both?